### PR TITLE
docs: Update Playwright config example

### PR DESCRIPTION
**Pull request by rstacruz**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- Updates the example config to use `defineConfig` as suggested in https://playwright.dev/docs/test-configuration.

- Updates the base URL to http://localhost:3000/ instead of http://localhost:3000/app/, which might represent a more typical case.

### CODE REVIEW

**Comment by sarah11918** [comment]
Thanks for this PR! I can see in the Playwright documentation they do have a different example config. I'm going to ping @chuckcarpenter since this was their original content based on their custom setup for a second pair of eyes here!

Chuck, any chance you can confirm that you've made these updates to your own setup, too? Or whether this is better/more recent general advice?

File: src/content/docs/en/guides/testing.mdx [hunk 1]
```diff
@@ -231,20 +231,20 @@ Here is an example of the configuration and commands required when using Yarn:
...
 
-    ```js title="playwright.config.ts" ins={3-8} "yarn preview"
-    import type { PlaywrightTestConfig } from '@playwright/test';
-    const config: PlaywrightTestConfig = {
+    ```js title="playwright.config.ts" ins={4-9} "yarn preview"
+    import { defineConfig } from '@playwright/test';
+    
+    export default defineConfig({
       webServer: {
         command: 'yarn preview',
```
Comment (ElianCodes) [inline src/content/docs/en/guides/testing.mdx:239]: @sarah11918, not entirely related to this PR, but do we 'recommend' `yarn` to run the commands here? Or should we just fall to PNPM or NPM which we use most in docs (imo)

Comment (sarah11918) [reply src/content/docs/en/guides/testing.mdx:239]: Great call out, Elian! This advanced user case was submitted by @chuckcarpenter as the setup they used. I know earlier in the page, for basic usage, we do include our normal package manager tabs for options. We do typically use NPM generically, all other things being equal. So I'd certainly be open to that change!

**Comment by ElianCodes** [comment]
@sarah11918 if it helps, I've verified the suggested changes in my own playwright config on elian.codes. The suggested changes are all valid!

This is my config:

```ts
import { defineConfig } from '@playwright/test';

export default defineConfig({
  testDir: './tests',
  use: {
    baseURL: 'http://localhost:3000',
  },
  outputDir: 'test-results/',
  webServer: {
    command: 'pnpm preview',
    port: 3000,
  },
});

```

**Comment by chuckcarpenter** [comment]
@sarah11918 sorry for the late response. Yeah, this LGTM based on changes I've also made on projects.

File: src/content/docs/en/guides/testing.mdx (hunk 1 above)
Comment (sarah11918) [reply src/content/docs/en/guides/testing.mdx:239]: @ElianCodes Would you be willing to make the suggested change here so I can just commit it and not have to go through the effort of manually adding you as a co-author to this PR?  :sweat_smile:

File: src/content/docs/en/guides/testing.mdx [hunk 2]
```diff
@@ -231,20 +231,20 @@ Here is an example of the configuration and commands required when using Yarn:
 1. Add a test script to your `package.json` file in the project root, such as `"test:e2e": "playwright test"`.
 1. In `playwright.config.ts`, add the `webServer` object and update the command value to `yarn preview`.
 
-    ```js title="playwright.config.ts" ins={3-8} "yarn preview"
-    import type { PlaywrightTestConfig } from '@playwright/test';
-    const config: PlaywrightTestConfig = {
+    ```js title="playwright.config.ts" ins={4-9} "yarn preview"
```
Comment (ElianCodes) [inline src/content/docs/en/guides/testing.mdx:234]:
```suggestion
    ```js title="playwright.config.ts" ins={4-9} "npm run preview"
```

File: src/content/docs/en/guides/testing.mdx [hunk 3]
```diff
@@ -231,20 +231,20 @@ Here is an example of the configuration and commands required when using Yarn:
...
       use: {
-        baseURL: 'http://localhost:3000/app/',
+        baseURL: 'http://localhost:3000/',
       },
-    };
-    export default config;
+    });
     ```
 
 1. Run `yarn build`, then run `yarn test:e2e` to run the Playwright tests.
```
Comment (ElianCodes) [inline src/content/docs/en/guides/testing.mdx:250]:
```suggestion
1. Run `npm run build`, then run `npm run test:e2e` to run the Playwright tests.
```

**Review by ElianCodes — CHANGES_REQUESTED** [review]
Left changes to switch from yarn to npm

File: src/content/docs/en/guides/testing.mdx [hunk 4]
```diff
@@ -229,25 +229,25 @@ You can also have Playwright start your server when you run your testing script
 Here is an example of the configuration and commands required when using Yarn:
```
Comment (ElianCodes) [inline src/content/docs/en/guides/testing.mdx:229]:
```suggestion
Here is an example of the configuration and commands required when using npm:
```

**Comment by sarah11918** [comment]
Looking great, gonna swap a couple more yarns that I think are still hanging around, then it's in the merge queue!

Thanks @rstacruz for this helpful update! We appreciate you taking the time to submit this PR!

Kind thanks to @chuckcarpenter and @ElianCodes for jumping in and confirming here. Go Team!

**Pull request merged**